index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>Document</title>
+    </head>
+
+    <body>
+
+        <header id="header">
+            <h1>Original Trombones</h1>
+            <nav id="nav-bar">
+
+                <a class="nav-link" href="">Features</a>
+                <a class="nav-link" href="">How it Works</a>
+                <a class="nav-link" href="">Pricing</a>
+
+            </nav>
+        </header>
+
+
+        <h1 class="secondtitle">Handcrafted, home-made masterpieces</h1>
+        <form id="form" action="">
+            <input type="email" name="email" id="email" placeholder="Enter Your email">
+            <input type="button" value="Get Started">
+        </form>
+
+        <section class="middle">
+            <h2> Premium Materials</h2>
+    
+        
+            <h2> Fast Shipping </h2>
+        
+        
+            <h2> Quality Assurance </h2>
+        </section>
+
+        <video src="https://www.freecodecamp.com/email-submit"></video>
+
+        <section class="mycard">
+            <div class="card">
+                <div class="title">
+                    <h1>TENOR TROMBONE</h1>
+                </div>
+                <h1 class="price">$600</h1>
+                <p class="text">Lorem ipsum.<br>
+                    Lorem ipsum.<br>
+                    Lorem ipsum dolor.<br>
+                    Lorem ipsum.</p>
+                <button class="btn">Select</button>
+            </div>
+
+            </div>
+        
+
+        
+            <div class="card">
+                <div class="title">
+                    <h1>BASS TROMBONE</h1>
```

Comment:
Kindly, use an `<h2>` to`<h6>`tag because we use `<h1>` for the most important text of the page and the site should have one `<h1>`.
`<h2>` should be better